src/dispatch/case/views.py

```diff
@@ -188,6 +189,35 @@ def create_case(
     return case
 
 
+@router.post(
+    "/{case_id}/resources/conversation",
+    response_model=CaseRead,
+    summary="Creates conversation channel for an existing case.",
+)
+def create_case_channel(
+    db_session: DbSession,
+    case_id: PrimaryKey,
+    current_case: CurrentCase,
+    background_tasks: BackgroundTasks,
+):
+    """Creates conversation channel for an existing case."""
+
+    case = get(case_id=case_id, db_session=db_session)
```

Comment:
question: is it necessary to refetch the case? i think `CurrentCase` should supply it:

```python
def get_current_case(db_session: DbSession, request: Request) -> Case:
    """Fetches a case or returns an HTTP 404."""
    case = get(db_session=db_session, case_id=request.path_params["case_id"])
    if not case:
        raise HTTPException(
            status_code=status.HTTP_404_NOT_FOUND,
            detail=[{"msg": "The requested case does not exist."}],
        )
    return case


CurrentCase = Annotated[Case, Depends(get_current_case)]
```

I could be missing a reason to refetch though as I see that happen across the codebase quite often.

Reply:
Done.